news/4.35/platform.html

```diff
@@ -97,6 +97,46 @@ <h2>General Updates </h2>
         </p>
       </td>
   </tr>
+  <tr id="rescaleOnRuntimePreference"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/pull/2461 -->
+    <td class="title">Enhanced Monitor-Specific UI Scaling (Experimental, Windows only)</td>
+    <td class="content">
+      <p>
+        In the previous release, a preference to enable an improved, monitor- and resolution-specific UI scaling on
+        Windows
+        was introduced (see <a href="../4.34/platform.php#rescale-on-runtime-preference">this news</a>). The feature
+        has been largely improved since then.
+        The feature makes each window adapt its scaling to the monitor it is currently placed on in a sharp,
+        resolution-specific way and without requiring the application to restart.
+        When using multiple windows, each of them will adapt its scaling to the monitor it is placed on.
+        When enabling the feature, it replaces the existing scaling support for high-resolution monitors that is limited
+        to initializing the application's window according to the scaling of the primary monitor at startup and adapting
+        to monitors with other scale values only via blurry scaling or via an application restart.
```

Comment:
I found this sentence a bit difficult to read (my personal experience). How about:

```suggestion
        When enabled, this new feature replaces the current scaling support for high-resolution monitors, which scales the application's window according to the scaling of the primary monitor upon initialization and either produces a blurry scaling when moving the window to another monitor or may ask for an application restart to avoid the blurriness.
```

?

Reply:
Agreed, the previous sentence was really hard to read. I took your proposal with slight adaptations, thank you!